### PR TITLE
Add script to create tables for a specific theme

### LINKED ISSUE #1341: Add script for creating tables per theme
In version 1.x we have a script `create_theme_tables` to create all tables for a specific theme. This script should be added again using the new model factories.

### PR DESCRIPTION
**Pull request by kdeininger**
Closes #1341

### CODE REVIEW

File: pyramid_oereb/standard/create_tables.py [hunk 1]
```diff
@@ -124,3 +137,85 @@ def create_standard_tables():
...
+
+    if Config.get_config() is None:
+        Config.init(
+            options.configuration,
+            options.section,
+            options.c2ctemplate_style
+        )
+
+    theme_config = None
+    for cfg in Config.get('plrs'):
```
Comment (vvmruder) [inline pyramid_oereb/standard/create_tables.py:202]: Maybe use direct method of Config object here: https://github.com/openoereb/pyramid_oereb/blob/2861ef390bb7eae7a4f42df62086be4ffa373f60/pyramid_oereb/lib/config.py#L815

Comment (vvmruder) [reply pyramid_oereb/standard/create_tables.py:202]: Config object got really handy ;)

File: pyramid_oereb/standard/create_tables.py [hunk 2]
```diff
@@ -12,6 +12,32 @@
 log = logging.getLogger(__name__)
 
 
+def create_theme_tables_(theme_config, tables_only=False, sql_file=None):
```
Comment (vvmruder) [inline pyramid_oereb/standard/create_tables.py:15]: Good idea to morph out of iteration to a function :)

File: pyramid_oereb/standard/create_tables.py (hunk 1 above)
Comment (kdeininger) [reply pyramid_oereb/standard/create_tables.py]: Thanks for the hint :+1: 
I've applied the method.

**Review by vvmruder — APPROVED** [review]
Happy

**Pull request merged**